src/viam/app/data_client.py

```diff
@@ -255,10 +255,14 @@ async def tabular_data_by_sql(self, organization_id: str, sql_query: str) -> Lis
 
     async def tabular_data_by_mql(self, organization_id: str, mql_binary: List[bytes]) -> List[Dict[str, ValueTypes]]:
         """Obtain unified tabular data and metadata, queried with MQL.
+        You must `import bson` to create valid bson binary objects as input for the `tabular_data_by_mql` method.
```

Comment:
Again, I don't want to prescribe a library, particularly given that the `bson` library hasn't been updated in 3 years. Further, a library isn't even technically required -- you could encode the bson by hand.

My suggestion would be to remove the `must`, and then provide an example of using a library:

```python3
async def tabular_data_by_mql(...):
    """Obtain unified tabular data and metadata, queried with MQL.
    `mql_binary` is an array of encoded `bson` queries. You can encode your bson queries using a library like `pymongo` or `bson`.

    ::
        # using bson 
        import bson
        data = ...(..., mql_binary=[bosn.dumps(...)])

        # using pymongo 
        import bson
        data = ...(..., mql_binary=[bosn.encode(...)])
"""

Reply:
You didn't actually say that before. I'm ok using your proposal.